src/chembl_downloader/downloader_checker.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+
+"""Codebase for logging and checking the working of chembl_downloader."""
+
+from typing import List, Tuple
+from tqdm import tqdm
+import chembl_downloader
+import bioversions
+import ftplib
+
+from tabulate import tabulate
+
+
+def get_version() -> List[str]:
+    """Get all version of ChEMBL."""
+
+    try:
+        latest_chembl_version = bioversions.get_version("chembl")
```

Comment:
this is a function inside `chembl_downloader`  - better to call it directly instead of importing another package

Reply:
Oh true. I will change that.

Reply:
Done in 027cea8